src/notes/StructuredFieldPlugin.jsx

```diff
@@ -339,12 +339,9 @@ function StructuredFieldPlugin(opts) {
                 }
             } else if (node.type === 'structured_field') {
                 const shortcut = node.data.shortcut;
-                if (shortcut instanceof InsertValue) {
-                    // Inserters have characters as their children. Use characters to get current text in the node.
-                    result += shortcut.getResultText(node.nodes[0].characters.map(c => c.text).join(''));
-                } else {
-                    result += shortcut.getResultText();
-                }
+                // TODO: Refactor to not need slate node text passed as an argument. This is currently used to reload the correct text on edited shortcuts. Refactor should result in no arguments passed.
+                const textToSerialize = (shortcut instanceof InsertValue && shortcut.metadata.isEditable) ? node.nodes[0].characters.map(c => c.text).join('') : undefined;
```

Comment:
[MG2] We should remove the `&& shortcut.metadata.isEditable`. Originally the idea with these was to only pass in this argument to editable shortcuts, but upon testing I realized that this should apply to all multiline inserters regardless of whether or not they are editable. I'm pretty sure I actually told Jules to put in editable so that's my bad.

A way to see why this is bad is to insert @active medications and then close and reopen the note